### ISSUE TITLE
Chant admin "Manuscript full text std spelling" widget is small

### ISSUE THREAD

**Issue by lucasmarchd01** [DDMAL/CantusDB#1065]
The Manuscript full text std spelling widget should be the same size as the Manuscript full text and Manuscript syllabized full text widgets below in the Admin chant detail page.

![image](https://github.com/DDMAL/CantusDB/assets/71031342/ce781942-e762-435b-8f6a-84185bc9e657)
